tools/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -222,12 +222,21 @@ export class FluidPackageCheck {
             // prepack scripts
             const prepack: string[] = [];
 
+            let concurrentBuildCompile = true;
+
             const buildPrefix = pkg.getScript("build:genver") ? "npm run build:genver && " : "";
             if (pkg.getScript("tsc")) {
                 if (pkg.getScript("build:test")) {
-                    buildCommonJs.push("tsc");
-                    buildCommonJs.push("build:test");
-                    buildCompile.push("build:commonjs");
+                    if (pkg.getScript("build:esnext")) {
```

Comment:
i don't really understand what's happening here, why do we care about build:esnext?

Reply:
There is a package that doesn't build for commonjs.  So I use the existence of build:esnext to tell me whether it build two ways.  I have added comment.